README.md

```diff
@@ -1,23 +1,21 @@
-AI CoE Demo Repo for OS-Climate
-==============================
+# Using Open Data Hub toolkit and Operate First infrastructure for OS-Climate
 
-This repository is the central location for the demos the AI CoE team is developing within the [OS-Climate](https://github.com/os-climate) project.
+This repository is the central location for the demos the Open Services ( previously AICoE) team is developing within the [OS-Climate](https://github.com/os-climate) project.
 
-## Demo 1 - ETL & Dashboarding
+The following demos provide examples of how to use the tools available with [Open Data Hub](https://opendatahub.io/) toolkit on an [Operate First](https://www.operate-first.cloud/) cluster to perform ETL, create inference pipelines, create interactive dashboards and visualizations of our data.
 
-This demo provides notebooks and an Elyra pipeline that provide an example of how to use the tools available with [Open Data Hub](https://opendatahub.io/) on an [Operate First](https://www.operate-first.cloud/) cluster to perform ETL and create interactive dashboards and visualizations of our data.
+## [Demo 1 - ETL & Dashboarding](notebooks/demo1/README.md)
 
-* Ingest raw data into Trino ([Notebook](https://github.com/os-climate/aicoe-osc-demo/blob/master/notebooks/demo1-create-tables.ipynb))
-* Run a join against a different federated data table ([Notebook](https://github.com/os-climate/aicoe-osc-demo/blob/master/notebooks/demo1-join-tables.ipynb))
-* Collect the results ([Pipeline](https://github.com/os-climate/aicoe-osc-demo/blob/master/notebooks/demo1.pipeline))
-* Visualize it in Superset ([Dashboard](https://superset-secure-odh-superset.apps.odh-cl1.apps.os-climate.org/superset/dashboard/3/))
+* [Ingest raw data from S3 as tables on Trino](https://github.com/os-climate/aicoe-osc-demo/blob/master/notebooks/demo1/demo1-create-tables.ipynb)
+* [Run SQL queries from a Jupyter Notebook environment](https://github.com/os-climate/aicoe-osc-demo/blob/master/notebooks/demo1/demo1-join-tables.ipynb)
+* [Demo 1 Elyra Pipeline](https://github.com/os-climate/aicoe-osc-demo/blob/master/notebooks/demo1/demo1.pipeline)
```

Comment:
```suggestion
* [Ingest raw data from S3 as tables on Trino](notebooks/demo1/demo1-create-tables.ipynb)
* [Run SQL queries from a Jupyter Notebook environment](notebooks/demo1/demo1-join-tables.ipynb)
* [Demo 1 Elyra Pipeline](notebooks/demo1/demo1.pipeline)
```
Shall we use relative path links instead of github url's?

Reply:
The elyra pipeline wasnt rendering in the Jupyterbook, hence kept that specific link as absolute.

Reply:
Ah I see, no worries then :+1:

Reply:
will change the other ones. 👍